src/screens/DataTableColumns.js

```diff
@@ -46,8 +46,12 @@ const DataTableColumnsPage = () => (
             <Example>{`['id', 'name', 'age']`}</Example>
           </PropertyValue>
           <PropertyValue type="array of objects">
+            <Description>
+              An array of objects of CheckBox props excluding the 'checked'
```

Comment:
Wording suggestion: "An array of objects, where the object contains CheckBox props. The CheckBox `checked` property is excluded to allow DataTableColumns to manage the state."

@britt6612 Can you clarify what `property` section means? I'm wondering if we should document or exclude.

Reply:
yeah it took me a while to figure it out as well until I looked in the `index.d.ts` and saw 
```
  options: (string | { property: string; label: string })[];
```

Reply:
so Ill add this property into the docs as well

Reply:
also not sure about that naming `properties` and why it wasn't `value`

Reply:
Or if we don't want to support all CheckBox props, but we do want to allow `disabled` specifically, we should be more specific and just add that.

Reply:
Based on discussion with @jcfilben, we've decided to document `disabled` for now and add typescript support for it in Grommet. However, since use cases haven't introduced a need for any other CheckBox props yet, we won't support those in Typescript, prop types, or docs.

See: https://github.com/grommet/grommet/pull/7028

Reply:
I pushed a wording change to align with this.